test/qa.jl

```diff
@@ -0,0 +1,4 @@
+using MuladdMacro, Aqua
+@testset "Aqua" begin
+    Aqua.test_all(MuladdMacro; test_ambiguities=(recursive = false,))
```

Comment:
I'm sorry, I misremembered. It should just be
```suggestion
    Aqua.test_all(MuladdMacro; ambiguities=(recursive = false,))
```